CMSgov/mint-app
# [EASI-3598] Update How will you choose beneficiaries? options

# EASI-3598

## Changes and Description

- Updated copy 

## How to test this change

1. Find question `How will you choose beneficiaries? Select all that apply.`  in the Beneficiaries section
2. Check dropdown options have been changed



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
